### PR TITLE
[#2435][#2436] Add to My Resources button issues

### PR DESCRIPTION
**Pull request by Maurier**
- Fixes overlapping tooltips.
- Disables the button for Discoverable resources.

### CODE REVIEW

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by Maurier** [comment]
Ready for review. Can be tested at https://park.hydroshare.org/

File: theme/static/js/resourceLandingAjax.js [hunk 1]
```diff
@@ -17,25 +17,27 @@ function label_ajax_submit() {
...
                     $("#btnMyResources").removeClass("btn-resource-remove");
-                    $("#btnMyResources").attr("title", "Add to my resources");
+                    $("#btnMyResources").attr("data-original-title", "Add to my resources");
                 }
+                $("#btnMyResources").tooltip('show')
             }
         },
         error: function (XMLHttpRequest, textStatus, errorThrown) {
-
+            console.log(errorThrown);
```
Comment (aphelionz) [inline theme/static/js/resourceLandingAjax.js]: Is this an error to be shown to the user? If so you can use the universalMessage popup

Comment (Maurier) [reply theme/static/js/resourceLandingAjax.js]: This one is not an error for the user, but for the developers. It only shows up in the browser's console.

Comment (aphelionz) [reply theme/static/js/resourceLandingAjax.js]: Ok, then I approve

**Pull request merged**